CHANGELOG.md

```diff
@@ -125,6 +28,20 @@ New types that wrap native types from Apple, Google and Amazon, and we cleaned u
 | `Purchases.RestoreTransactions` | `Purchases.restorePurchases` |
 | `Purchases.GetPaymentDiscount` | `Purchases.getPromotionalOffer` |
 | `Purchases.UpdatedPurchaserInfoListener` | `Purchases.updatedCustomerInfoListener` |
+| `Purchases.getProductInfo` | `Purchases.getProducts` |
+| `Purchases.setup` | `Purchases.configure` |
+| `PurchasesStoreProduct.intro_price` | `PurchasesStoreProduct.introPrice` |
+| `PurchasesStoreProduct.price_string` | `PurchasesStoreProduct.priceString` |
+| `PurchasesStoreProduct.currency_code` | `PurchasesStoreProduct.currencyCode` |
+
+### Bugfixes
+* Configure should be called in the current thread and not in a UI thread (#193) via Cesar de la Vega (@vegaro)
+### New Features
+* Add support for Firebase, Mixpanel, and CleverTap integrations (#177) via Joshua Liebowitz (@taquitos)
+* Apple AdServices support (#167)
+### Other Changes
+* Improve purchase tester (#186) via Andy Boedo (@aboedo)
+* cordova-plugin-purchases requires Xcode 13.0+ and minimum targets iOS 11.0+ and macOS 10.13+ (#160) via NachoSoto (@NachoSoto)
 
```

Comment:
I know we already link to the full changelog at the beginning, but maybe here we can mention something like "lots of other under-the-hood improvements" and link to it too?

Reply:
Yes, I like it, let's do it